### DISCUSSION TITLE
Build

### DISCUSSION THREAD

**Discussion by JulsKawa** [cagnulein/qdomyos-zwift#213] [Q&A]
Hello , i try to build the android version of the app but it doesn't work.

2: Task failed with an exception.
-----------
* What went wrong:
A problem occurred configuring root project 'android'.
> compileSdkVersion is not specified. Please add it to build.gradle

**Comment by cagnulein** [comment]
did you install the toolchain of android?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 22 mar 2021 alle ore 15:03 JulsKawa ***@***.***>
ha scritto:
